charts/pulsar/values.yaml

```diff
@@ -806,6 +806,7 @@ broker:
     maxReplicas: 3
     metrics: ~
     behavior: ~
+  podManagementPolicy: Parallel
```

Comment:
simply provide the key without the value, so that it's absent by default.
Please add a short documentation to the key. It's worth mentioning that the value cannot be modified without manual operations after a cluster has been deployed.